include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -65,7 +65,22 @@ class MGConstrainedDoFs : public Subscriptor
   template <int dim, int spacedim>
   void initialize(const DoFHandler<dim,spacedim> &dof,
                   const typename FunctionMap<dim>::type &function_map,
-                  const ComponentMask &component_mask = ComponentMask());
+                  const ComponentMask &component_mask = ComponentMask()) DEAL_II_DEPRECATED;
+
+  /**
+   * Fill the internal data structures with information
+   * about Dirichlet boundary dofs.
+   *
+   * The initialize() function has to be called additionally
+   * to set hanging node constraints.
```

Comment:
At the moment, the only purpose of  `initialize` is to set up the refinement_edge_indices. Therefore there is no separate function `set_hanging_node_constraints`.